README.md

```diff
@@ -52,15 +52,12 @@ django-admin-interface is a modern **responsive flat admin interface customizabl
 INSTALLED_APPS = (
     #...
     "admin_interface",
-    "flat_responsive", # only if django version < 2.0
-    "flat", # only if django version < 1.9
```

Comment:
Should add a migration note telling people to remove these?

Reply:
mhmh no... even if they are installed nothing happens.